<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrgDialog</class>
 <widget class="QWidget" name="OrgDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1169</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="img">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>0</y>
     <width>1211</width>
     <height>701</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/dashboard/dashboard.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="editorg">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(38, 0, 115);
background-color: rgb(216, 213, 184);
border-radius:6px;</string>
   </property>
   <property name="text">
    <string> Edit Organization</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteorg">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(38, 0, 115);
background-color: rgb(216, 213, 184);
border-radius:6px;</string>
   </property>
   <property name="text">
    <string>Delete Organization</string>
   </property>
  </widget>
  <widget class="QComboBox" name="roleComboBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>330</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:5px;
background-color: rgb(140, 140, 140);</string>
   </property>
   <item>
    <property name="text">
     <string notr="true">Role</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Admin</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>User</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="idLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Members  :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="idLineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:5px;
background-color: rgb(181, 181, 181);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_user">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>410</y>
     <width>181</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(241, 238, 205);
background-color: rgb(113, 113, 113);
border-radius:6px;</string>
   </property>
   <property name="text">
    <string>Delete User</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addOrEditUser">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>370</y>
     <width>181</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(241, 238, 205);
background-color: rgb(113, 113, 113);
border-radius:6px;</string>
   </property>
   <property name="text">
    <string>Add or Edit User</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 157);
background-color: rgb(241, 238, 205);
border-radius:7px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="descriptionLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 157);
background-color: rgb(241, 238, 205);
border-radius:7px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>100</y>
     <width>761</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(26, 17, 68);
border-color: rgb(26, 17, 68);
</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="Projects">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(26, 17, 68);</string>
    </property>
    <attribute name="title">
     <string>Projects</string>
    </attribute>
    <widget class="QPushButton" name="addproject">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>220</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(140, 140, 140);
border-radius:6px;</string>
     </property>
     <property name="text">
      <string>Add project</string>
     </property>
    </widget>
    <widget class="QLabel" name="projectlabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Projects :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>120</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(248, 245, 211);</string>
     </property>
     <property name="text">
      <string>Name :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>170</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(248, 245, 211);</string>
     </property>
     <property name="text">
      <string>Description :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="project_name_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>120</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(199, 199, 199);
border-radius:7px;</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="projectdes_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>170</y>
       <width>161</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(199, 199, 199);
border-radius:7px;</string>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>310</y>
       <width>651</width>
       <height>161</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="prolay"/>
    </widget>
   </widget>
   <widget class="QWidget" name="Teams">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(26, 17, 68);</string>
    </property>
    <attribute name="title">
     <string>Teams</string>
    </attribute>
    <widget class="QLabel" name="teamlabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Teams :</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addteam">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>200</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(140, 140, 140);
border-radius:6px;</string>
     </property>
     <property name="text">
      <string>Add team</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="team_name">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>100</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(199, 199, 199);
border-radius:7px;</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="teamdes">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>150</y>
       <width>161</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(199, 199, 199);
border-radius:7px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(248, 245, 211);</string>
     </property>
     <property name="text">
      <string>Name :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>150</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(248, 245, 211);</string>
     </property>
     <property name="text">
      <string>Description :</string>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>330</y>
       <width>661</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="teamlay"/>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
